src/lib/y2partitioner/widgets/overview.rb

```diff
@@ -14,10 +15,15 @@ module Widgets
     # TODO: abstract treewidget from it
     class Overview < CWM::CustomWidget
       # creates new widget for given device graph
-      def initialize(device_graph)
+      # @param [Y2Storage::Devicegraph] device_graph
+      # @param [CWM::ReplacePoint] details_rp replace-point for the details pane
+      def initialize(device_graph, details_rp:)
+        textdomain "storage"
         self.handle_all_events = true
+        @hostname = Yast::Hostname.CurrentHostname
```

Comment:
Currently just nitpicking, but it's not super-good when constructor contains calls that might take more time (I haven't checked Yast::Hostname.CurrentHostname code though). Ideally UI starts ASAP and most of reading is done with lazy loading. Anyway, depends on you.